src/CanvasView.tsx

```diff
@@ -127,10 +127,10 @@ export const CanvasView: React.FC = () => {
               variant="secondary"
             />
             <IconButton
-              aria-label="Fit to view"
-              title="Fit to view"
+              aria-label="Fit to content"
+              title="Fit to content"
               icon={<CompressIcon />}
-              onClick={() => canvasService.send('FIT_TO_VIEW')}
+              onClick={() => canvasService.send('FIT_TO_CONTENT')}
```

Comment:
why are we changing this term?

Reply:
https://github.com/statelyai/registry/pull/196#discussion_r747213455

Reply:
This is the one that The Piano Man prefers